app/cdap/components/GridTable/components/NestedMenu/menuOptions/datatypeOptions.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import T from 'i18n-react';
+import { DATA_TYPE_LABEL_PREFIX } from 'components/GridTable/components/NestedMenu/menuOptions/constants';
+import { TOOLBAR_ICONS_LABEL_ALL_PREFIX } from 'components/GridTable/components/TransformationToolbar/constants';
+import { IMenuItem } from '../../MenuItemComponent/types';
+
+export const DATATYPE_OPTIONS: IMenuItem[] = [
+  {
```

Comment:
since this array is so simple you could make it like this? But also... all supported data types are just a prefix?
```
['string', 'boolean'...].map((dataType) => {
  return {
    value: dataType,
    label: T.translate(`${DATA_TYPE_LABEL_PREFIX}.${dataType}),
    supportedDataType: [TOOLBAR_ICONS_LABEL_ALL_PREFIX]
  }
);
```

Reply:
Done